js/components/pages/PassagePage.react.js

```diff
@@ -112,6 +112,8 @@ export class PassagePage extends Component {
           </div>
 
           <div className="meta-information">
+            <a className="left" href="http://www.esv.org">ESV</a>
+            <a className="right" href="#">Help</a>
```

Comment:
I left the href blank since it sounded like we might be uploading the instructions page to the server separately. (And also after fighting with webpack for a good hour I couldn't get it to pull a static html page into the build.)